native/cocos/editor-support/spine-wasm/spine-type-export.cpp

```diff
@@ -347,60 +564,45 @@ EMSCRIPTEN_BINDINGS(spine) {
     class_<Interpolation>("Interpolation")
         .function("apply", &Interpolation::apply, pure_virtual());
 
-    class_<Triangulator>("Triangulator")
-        .constructor<>()
-        .function("triangulate", &Triangulator::triangulate)
-        .function("decompose", &Triangulator::decompose, allow_raw_pointers());
+    class_<HasRendererObject>("HasRendererObject")
+        .constructor<>();
+
+    // class_<Triangulator>("Triangulator")
+    //     .constructor<>()
+    //     .function("triangulate", &Triangulator::triangulate)
+    //     .function("decompose", &Triangulator::decompose, allow_raw_pointers());
 
     class_<ConstraintData>("ConstraintData")
         .constructor<const String &>()
-        .function("getName", optional_override([](ConstraintData &obj) { return STRING_SP2STD(obj.getName()); }))
-        .function("getOrder", &ConstraintData::getOrder)
-        .function("setOrder", &ConstraintData::setOrder)
-        .function("getSkinRequired", &ConstraintData::isSkinRequired)
-        .function("setSkinRequired", &ConstraintData::setSkinRequired);
+        .property("name", &ConstraintData::getName)
+        .property("order", &ConstraintData::_order)
+        .property("skinRequired", &ConstraintData::_skinRequired);
```

Comment:
Use `.property` instead of `.function` will make binary size smaller and make spine-define.ts smaller.